archey3

```diff
@@ -802,7 +802,6 @@ class Archey(object):
                 args = ()
 
             yield groups["func"], args
-        raise StopIteration
```

Comment:
I would argue that this shouldn't be removed, but just changed to `return`

Reply:
im not keen with python, so let author decide

Reply:
I think this is fine.